depends/packages/qt.mk

```diff
@@ -126,6 +126,15 @@ $(package)_config_opts_darwin += -device-option MAC_SDK_VERSION=$(OSX_SDK_VERSIO
 $(package)_config_opts_darwin += -device-option CROSS_COMPILE="$(host)-"
 $(package)_config_opts_darwin += -device-option MAC_TARGET=$(host)
 $(package)_config_opts_darwin += -device-option XCODE_VERSION=$(XCODE_VERSION)
+else
+# Qt must be able to build its own tools for the build platform.
```

Comment:
I'm having trouble parsing this comment. Can you describe the actual problem? What about the custom mkspec is the issue?